config/routes.rb

```diff
@@ -6,6 +6,7 @@
   post 'comment/create/token/:id.:format', to: 'comment#create_by_token'
 
   get 'searches/test' => 'searches#test'
+  post '/request_title_change/:id/:title' => 'notes#request_title_change'
```

Comment:
Maybe we could simplify this and also not put the title itself in the path, as it may contain punctuation? It could be using `?title=____` instead.

Reply:
Sure, we could do the same with id!! Also we'd need to have a check in the request_title_update method, since the route could easily be manipulated to change the id or title so an authorisation check is a must at the back-end.